frontend/src/components/TransactionLinks/TransactionLink.vue

```diff
@@ -96,6 +117,15 @@ export default {
     decay() {
       return `${this.amount - this.holdAvailableAmount}`
     },
+    link() {
+      return `${window.location.origin}/redeem/${this.code}`
+    },
```

Comment:
duplicate code? Note the conflict with the PR of @Tirokk

Reply:
nein. wird an verschiedenen stellen verwendet